src/plugins/conductor/TimeConductorService.js

```diff
@@ -0,0 +1,170 @@
+         * from the value provided by a tick source to determine the start
+         * bound.
+         * @property {number} end Used to set the end bound of the
+         * TimeConductor on tick. A positive value that will be added
+         * from the value provided by a tick source to determine the start
+         * bound.
+         */
+        /**
+         * Deltas define the offset from the latest time value provided by
+         * the current tick source. Deltas are only valid in realtime or LAD
+         * modes.
+         *
+         * Realtime mode:
+         *     - start: A time in ms before now which will be used to
+         *     determine the 'start' bound on tick
+         *     - end: A time in ms after now which will be used to determine
+         *     the 'end' bound on tick
+         *
+         * LAD mode:
+         *     - start: A time in ms before the timestamp of the last data
+         *     received which will be used to determine the 'start' bound on
+         *     tick
+         *     - end: A time in ms after the timestamp of the last data received
+         *     which will be used to determine the 'end' bound on tick
+         * @returns {TimeConductorDeltas} current value of the deltas
+         */
+        TimeConductorService.prototype.deltas = function () {
+            // Get / Set deltas
+        };
+
+        /**
+         * Availability of modes depends on the time systems and tick
+         * sources available. For example, Latest Available Data mode will
+         * not be available if there are no time systems and tick sources
+         * that support LAD mode.
+         * @returns {ModeMetadata[]}
+         */
+        TimeConductorService.prototype.availableTickSources = function () {
+            var timeAPI = this.timeAPI;
+            //Return all tick sources
+            return _.uniq(this.config.map(function (option) {
+                return option.tickSource && timeAPI.tickSources(option.tickSource);
+            }.bind(this)));
+        };
+
+        TimeConductorService.prototype.availableTimeSystems = function () {
+            return Object.values(this.timeAPI.timeSystems);
+        };
+
+        /**
+         * An event to indicate that zooming is taking place
+         * @event platform.features.conductor.TimeConductorService~zoom
+         * @property {ZoomLevel} zoom the new zoom level.
+         */
+        /**
+         * Zoom to given time span. Will fire a zoom event with new zoom
+         * bounds. Zoom bounds emitted in this way are considered ephemeral
+         * and should be overridden by any time conductor bounds events. Does
+         * not set bounds globally.
+         * @param {number} zoom A time duration in ms
+         * @fires platform.features.conductor.TimeConductorService~zoom
+         * @see module:openmct.TimeConductor#bounds
+         */
+        TimeConductorService.prototype.zoom = function (timeSpan) {
```

Comment:
Yeah good question. This is replicating functionality already in the `TimeConductorViewService`  for synchronization between some of the conductor view elements. Zoom is treated differently to bounds changes to allow immediate scaling of the conductor ticks and the start and end date fields. A bounds change event is only triggered on mouse up. The intention is not to provide this to other views outside of the time conductor, but I can see how it might potentially be useful in some cases. Eg. having realtime feedback in plots of the zoom and pan state would be great, even if the data isn't available yet.

I'm reluctant to punt it just because it's necessary infrastructure for the Time Conductor UI, but if we move the ticking stuff into the public API, maybe we could restrict the scope of this service to just the conductor view elements again.